src/stretch/motion/constants.py

```diff
@@ -161,7 +161,7 @@ class HelloStretchIdx:
         -1.5,  # wrist pitch
         0.0,  # wrist yaw
         0.0,
-        math.radians(-65),
+        math.radians(-50),
```

Comment:
this needs to be synced across remote and local - what's the justification behind the change?

Reply:
This change is no longer needed with my new robot bounds filter

Reply:
ok then we should revert it